projects/plugins/jetpack/_inc/lib/class-jetpack-podcast-helper.php

```diff
@@ -277,6 +312,13 @@ public function load_feed( $force_refresh = false ) {
 		// Add action: detect the podcast feed from the provided feed URL.
 		add_action( 'wp_feed_options', array( __CLASS__, 'set_podcast_locator' ) );
 
+		$cache_timeout_filter_added = false;
+		if ( $this->cache_timeout !== null ) {
+			// If we have a custom cache timeout, apply the custom timeout value.
+			add_filter( 'wp_feed_cache_transient_lifetime', array( $this, 'filter_podcast_cache_timeout' ) );
```

Comment:
I'm wondering if it makes sense to use something other than the default priority here?

Since this now only affects the cache timeout when explicitly set, it might make sense to have a higher priority so it takes precedence over any "generic" filters for `wp_feed_cache_transient_lifetime` that are be present.

e.g.
```php
add_filter( 'wp_feed_cache_transient_lifetime', array( $this, 'filter_podcast_cache_timeout' ), 20 );
```

Reply:
Good thought. I've shifted the priority back to 20 so it executes later in 782639b.